Update PCI DSS responsibility matrix (#1405)
platformsh/platformsh-docs: docs/src/security/compliance-guidance.md

* **Backups** - On Platform.sh Professional customers are responsible for all application and database backups.

> The following spreadsheet provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions or assurances from Platform.sh, its affiliates, vendors or licensors.

Reviewer: ```suggestion
The [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx) provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions or assurances from Platform.sh, its affiliates, vendors or licensors.
```

> The [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx) provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions or assurances from Platform.sh, its affiliates, vendors or licensors.

Reviewer: ```suggestion
The [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx) provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions, or assurances from Platform.sh, its affiliates, vendors, or licensors.
```

> The [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx) provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions, or assurances from Platform.sh, its affiliates, vendors, or licensors.

Reviewer: ```suggestion
The [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx) (Excel) provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This document was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions, or assurances from Platform.sh, its affiliates, vendors, or licensors.
```

> By opening the below Excel spreadsheet, you accept and agree to these Terms of Use. If you do not wish to adhere to these Terms of Use, do not open, download, save, or otherwise access the Excel spreadsheet. 

Reviewer: ```suggestion
By opening the linked document you accept and agree to these Terms of Use. If you do not wish to adhere to these Terms of Use, do not open, download, save, or otherwise access the Excel spreadsheet.
```

> Please click the following link to download the [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx).

Reviewer: ```suggestion
```

> Please click the following link to download the [Platform.sh PCI Responsibility Matrix](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ6iXmdeFoT2OycEEd1rOpxV7t5fdPqthommxk-wtp7mZTy0FDHIh7gXQdl1c-qtkl9Pqsduhbza3i3/pub?output=xlsx).

Reviewer: ```suggestion
```
